public/javascripts/movieInfo.js

```diff
@@ -131,7 +131,7 @@ function bookTicket() {
     var strUser = e.options[e.selectedIndex].value.toString();
 
     var formatDate = new Date(strUser);
-    var getHours = ('0'+formatDate.getHours()).substr(-2);;
```

Comment:
This just needs to be a hard coded "9:00"

You don't need to do any date formatting.